.chronus/changes/fix-examples-const-2024-8-30-17-28-17.md

```diff
@@ -0,0 +1,8 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: fix
+packages:
+  - "@typespec/compiler"
+---
+
+Fix: Passing `const` of model type to `@example`
```

Comment:
nit: add const example to [doc](https://typespec.io/docs/next/standard-library/built-in-decorators#@example) as well..